### DISCUSSION TITLE
part-ii/2019-130/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#230] [Question comments]
Part II 2019 Representation Theory: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2019-130/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

I did $\mathbb{R}G \cong \frac{\mathbb{R}[X]}{\left(X^n-1\right)}$, and decomposed the RHS using Chinese Remainder Theorem. 

</details>
